### PR TITLE
Assignment2

### PR DESCRIPTION
**Pull request by ReemaJiries**

### CODE REVIEW

File: Binary_Tree.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
+from Node import Node
+
+"""
+the Binary Tree consist of Nodes with no duplicates keys and root node.
+onstructor: accepts key of the root Node
```
Comment (ariava) [inline Binary_Tree.py]: Typo: onstructor -> constructor

File: Binary_Tree.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
+from Node import Node
+
+"""
```
Comment (ariava) [inline Binary_Tree.py]: Docstrings go just after the "def" line of the element they describe. See e.g. https://www.python.org/dev/peps/pep-0257/#one-line-docstrings.

**Review by ariava — COMMENTED** [review]
Great job with code, testing and documentation!

File: Binary_Tree.py (hunk 2 above)
Comment (ReemaJiries) [reply Binary_Tree.py]: Thank you, I have updated it.

**Review by ariava — APPROVED** [review]
Looks good!

**Pull request closed without merging**